### DISCUSSION TITLE
🐛 FEAT: Shortcuts on webapp

### DISCUSSION THREAD

**Discussion by HarryLesnar24** [dailydotdev/daily#1442] [Feature Requests]
### What went wrong? 🤔

Hi,
If I open new tab recently you added feature that show up the list of links that I frequently visiting in browser **Quick Link** but it had one problem when I open new tab it show but if I get into other options like settings page squads discover so on and then I try to get back home page there is multiple options there so I list out below what are those way to get back home

1. We just press back button in browser we get back to the previous state if I follow this way means the **Quick Link** there without any issue
2. Second method we can press daily.dev logo to get back home but here is the problem arise the **Quick Link** gone from main page if I want to get back means I need to open a new tab then only it show again
3. Third method we can directly press my feed option from anywhere to get back the home page here also same problem the **Quick Link** gone from home page

### Expected Behavior

if I go any option and then if I get back to home the **Quick Link** need to be there that's what the expected behaviour should like instead of it gone from home page

### Steps to Reproduce Issue

```bash
1. Go to any options like Notifications Discussion except `Explore` option because it behave what way I expected without any issue at all it retain the **Quick Link** when I get back to home in multiple way
2. Click on my feed option or daily.dev logo instead of back button option from browser
3. See the error the **Quick Link** option gone from home page
```

### Solution Proposed

I'm unable to paste screenshot in below option Screenshots so I pasted here for your reference

![image](https://github.com/user-attachments/assets/d8836ba1-d373-4d06-9a5e-eb7b2fd7d01a)

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Browsers

Chrome

### OS

Windows

### Version of daily.dev

3.34.1

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by nensidosari** [comment]
Hi @HarryLesnar24,

Thank you for reporting the issue!

We are aware of the problem, this happens because the shortcuts are only available on the extension, and once you move to any other pages that are not part of the extension every link you click after that won't take you back to extension, thus shortcuts won't be available. 

We will try to give a better solution to this problem as soon as we can. 

Best!
